### ISSUE TITLE
[Feature] Case-Insensitive Flag for RegEx Filters

### ISSUE THREAD

**Issue by SpedNSFW** [stashapp/stash#1111]
Pretty self-explanatory. I'd love to see an option for case-insensitive RegEx filtering.

**Comment by WithoutPants** [comment]
This should probably be addressed with a checkbox, but the workaround is to prefix the regex with `(?i)`.

**Comment by SpedNSFW** [comment]
> This should probably be addressed with a checkbox, but the workaround is to prefix the regex with `(?i)`.

Yeah I would have expected a checkbox. Was unaware of the `(?i)` though, RegEx is different in basically every language's implementation of it.
